### PR TITLE
[10.0] Migrate mrp_product_details

### PR DESCRIPTION
**Pull request by bimajatiwijaya**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please take the occasion to rename to singular: `mrp_bom_product_detail`

File: mrp_bom_product_detail/README.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================
+Bom product details
+===================
+
+This module adds information product price and product stock on bill of material tree view and bom component.
```
Comment (elicoidal) [inline mrp_bom_product_detail/README.rst]: s/bom/BOM

Comment (elicoidal) [reply mrp_bom_product_detail/README.rst]: In general try to keep the sentences around no more 80 characters (nice to have)

File: mrp_bom_product_detail/README.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+
+Configuration
+=============
+
+To show price and product stock on bill of material, please apply product variant feature.
```
Comment (elicoidal) [inline mrp_bom_product_detail/README.rst]: end the sentence with ":"

File: mrp_bom_product_detail/README.rst [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+1. Go to Manufacturing > Configuration > Settings > Product Variants
```
Comment (elicoidal) [inline mrp_bom_product_detail/README.rst]: better use the "#. " feature from rst for enumerated lines (nice to have)

File: mrp_bom_product_detail/README.rst [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+2. Then select "Products can have several attributes, defining variants (Example: size, color,...)" and click apply
+
+Usage
+=====
+
+To use this module, you need to:
+
+1. Go to Manufacturing > Master Data > Bill of Materials
+   a. Tree view bom and component bom will show cost and stock.
```
Comment (elicoidal) [inline mrp_bom_product_detail/README.rst]: the level is wrong, it should be something like:
```
#. Go to Manufacturing > Master Data > Bill of Materials
#. Tree view bom and component bom will show cost and stock.

File: mrp_bom_product_detail/__init__.py [hunk 5]
```diff
@@ -20,4 +20,4 @@
 #
 ##############################################################################
 
-import mrp_bom_product_details
```
Comment (elicoidal) [inline mrp_bom_product_detail/__init__.py]: Can you use short headers?

File: mrp_bom_product_detail/README.rst [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Bima Wijaya <bimajatiwijaya@gmail.com>
```
Comment (elicoidal) [inline mrp_bom_product_detail/README.rst]: Are you the only contributor to the module?

File: mrp_bom_product_detail/__manifest__.py [hunk 7]
```diff
@@ -21,16 +21,17 @@
...
     'license': 'GPL-3 or any later version',
     'category': 'Manufacturing',
-    'description': """
-    This module adds product price and stock to bom view
-    """,
-    'depends': ["base", "mrp"],
+    'depends': [
+        "mrp",
+    ],
     'demo': [],
```
Comment (elicoidal) [inline mrp_bom_product_detail/__manifest__.py]: remove empty keys

File: mrp_bom_product_detail/views/mrp_bom_product_details.xml [hunk 8]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
+<odoo>
```
Comment (elicoidal) [inline mrp_bom_product_detail/views/mrp_bom_product_details.xml:2]: You can add copyright notice to xml if you wish

**Review by elicoidal — CHANGES_REQUESTED** [review]

File: mrp_bom_product_detail/README.rst (hunk 1 above)
Comment (pedrobaeza) [reply mrp_bom_product_detail/README.rst]: Better BoM (Bill of Materials)

File: mrp_bom_product_detail/__init__.py (hunk 5 above)
Comment (bimajatiwijaya) [reply mrp_bom_product_detail/__init__.py]: i keep the header. Do you want to delete it ?

Comment (andhit-r) [reply mrp_bom_product_detail/__init__.py]: What @elicoidal means is you should use short header format according to OCA guidelines. See https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py header's as an example. Pay attention to first 3 lines.

**Review by andhit-r — COMMENTED** [review]
LGTM. Waiting for header to adjust according OCA guidelines.

**Comment by andhit-r** [comment]
@bimajatiwijaya i have no migrating experience yet. But i guess that change in module name will have an effect for old db that has this module installed. Won't we need migration script?

**Comment by pedrobaeza** [comment]
That's simple achieve adding the renaming on OpenUpgrade on the file apriori.py. I'll do it directly when this gets merged.

**Review by andhit-r — APPROVED** [review]
Nice. LGTM.

**Comment by LoisRForgeFlow** [comment]
@elicoidal could you update your review?

**Review by elicoidal — APPROVED** [review]
LGTM

**Review by LoisRForgeFlow — APPROVED** [review]
Code review LGTM

**Comment by LoisRForgeFlow** [comment]
@bimajatiwijaya Could you squash your commits? We'll merge after it

edit: also try to recover commit history: [migration wiki](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0)

**Comment by bimajatiwijaya** [comment]
@lreficent : i tried. i can't do squash. i create new pull request https://github.com/OCA/manufacture/pull/241

**Pull request closed without merging**